docs/reference/commandline/plugin_create.md

```diff
@@ -15,7 +15,7 @@ Create a plugin from a rootfs and configuration. Plugin data directory must cont
 ## Description
 
 Creates a plugin. Before creating the plugin, prepare the plugin's root
-filesystem as well as the [config.json](../../extend/config.md).
+filesystem as well as the [config.json](https://docs.docker.com/engine/extend/config/).
```

Comment:
These can't work in the new situation? (mostly looking because (unlike, e.g. "dockerfile" reference) this is a file in the same repository)